### PR TITLE
Refactor kms configuration

### PR DESCRIPTION
**Pull request by mattcollier**
- **BREAKING**: Remove config variables for `privateKmsBaseUrl` and
  `publicKmsBaseUrl`. The KMS is now configured via `bedrock-profile`.

### CODE REVIEW

File: test/test.config.js [hunk 1]
```diff
@@ -36,3 +36,6 @@ roles['bedrock-test.regular'] = {
     permissions.EDV_CONFIG_REMOVE.id
   ]
 };
+
+config.profile.kms.baseUrl = `${config.server.baseUri}/kms`;
+config.profile.kms.ipAllowList = ['127.0.0.1/32'];
```
Comment (dlongley) [inline test/test.config.js:41]: This is fine for now, but the plan is to move this into keystore configs, right?

**Review by dlongley — APPROVED** [review]

File: test/test.config.js (hunk 1 above)
Comment (mattcollier) [reply test/test.config.js:41]: this informs what ipAllowList gets into the keystore configs.  

See: https://github.com/digitalbazaar/bedrock-profile/pull/67

specifically: https://github.com/digitalbazaar/bedrock-profile/compare/refactor-kms?expand=1#diff-7f2cfbda4d88af2c71bb531dea727cc6cf1b9c6acdf67b2b96083075d7148f38R20-R39

Comment (dlongley) [reply test/test.config.js:41]: Aha! Thank you.

**Comment by dlongley** [comment]
@gannan08 -- can you confirm that this is out of date and should be closed now? If so, please close.

**Comment by dlongley** [comment]
Superseded by #40 and others, closing.

**Pull request closed without merging**